src/AccordionContainer/AccordionContainer.spec.tsx

```diff
@@ -122,11 +122,10 @@ describe('Accordion', () => {
         );
     });
 
-    it('adding an expanded item to a strict-accordion closes other items', () => {
+    it("adding an expanded item to an accordion that doesn't allow mutliple expansions closes other items", () => {
```

Comment:
Typo here: 'mutliple'

Reply:
Wow, I'm really having trouble with that word today!